### PR TITLE
[Re] Added Detecting and Removing Cycle from a linked list

### LINKED ISSUE #520: Linked List Trickiest Questions in C++
Hey...I want to add trickiest and mind boggling questions on linked list in C++.
For example - Converting singly linked list into circular linked list.
                       Removing loop from a linked list
                       Partitioning linked list around a given value
and lots more.

### PR DESCRIPTION
**Pull request by saumyashrm2802**
Fixes #520 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Sample Input / Output.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

#### Changes proposed in this pull request:

- Added C++ code for Detecting and removing Cycle from a linked list
(will add code for more trickiest questions soon)

#### Languages Used:
- C++ 

#### Files Added:
- Detecting and Removing Cycle in a Linked List .cpp

### CODE REVIEW

File: Linked List Trickiest Questions/Detecting and Removing Cycle in a Linked List.cpp [hunk 1]
```diff
@@ -0,0 +1,97 @@
+#include<iostream>
+#include<conio.h>
```
Comment (jainaman224) [inline Linked List Trickiest Questions/Detecting and Removing Cycle in a Linked List.cpp]: Conio is not supported by standard gcc compiler. Please remove it.

**Comment by saumyashrm2802** [comment]
@jainaman224 Sir, The file name is "Detecting and Removing Cycle from a linked list". May you please suggest any other name for the file, if this is not suitable?

**Comment by jainaman224** [comment]
Are you using Floyd's algorithm to detect cycle?

**Comment by marwahmanbir** [comment]
@saumyashrm2802 you can go for a shorter name like "Linked List Cycle Operations"
Kindly check the format for naming first, you can check how existing files and directories are named.

**Comment by MastersAbh** [comment]
Hi @saumyashrm2802 ,
still working on it ?

**Comment by jainaman224** [comment]
@marwahmanbir @MastersAbh Suggest some name. I will fix and merge this. It is delay on my end.

**Comment by MastersAbh** [comment]
Cycle_Detection inside Linked List folder would be fine. Removal can be omitted from file name. @jainaman224

**Comment by jainaman224** [comment]
Ok

**Pull request merged**